overlord/configstate/configcore/picfg.go

```diff
@@ -106,7 +107,7 @@ func (e *piConfigNotSupportedError) Error() string {
 	return fmt.Sprintf("configuring not supported: %s", e.reason)
 }
 
-var ignorePrefix = []byte("# SNAPD IGNORE\n")
+var reIgnorePrefix = regexp.MustCompile(`(?i)^#\s+Snapd-Edit:\s+no.*`)
```

Comment:
what's the intention to allow random things at the end?

I would have thought we would do "\s*$"  maybe?

if you think we should allow random things,  we probably want at least a space after the "no", no?

Reply:
Thanks! Sorry, I think I misunderstood the earlier suggestion, I will only allow whitespace after the stanza.